src/TermInterface.jl

```diff
@@ -1,18 +1,32 @@
 module TermInterface
 
 """
-  istree(x)
```

Comment:
Why was `istree` removed? There can be non-call tree terms! I've been using `istree` intensively and there's still a difference, i think that `iscall(x)` should imply `istree(x)`, but it should still be present

Reply:
Example `v[x, y, z]`

Reply:
NVM, just saw `isexpr`